logprep/processor/generic_adder/rule.py

```diff
@@ -132,5 +215,5 @@ def _check_if_valid(rule: dict):
                 if not isinstance(generic_adder_cfg[field], (str, list)):
                     raise InvalidGenericAdderDefinition(error_msg)
                 if isinstance(generic_adder_cfg[field], list):
-                    if not all([isinstance(item, str) for item in generic_adder_cfg[field]]):
+                    if not all((isinstance(item, str) for item in generic_adder_cfg[field])):
```

Comment:
I assume that PyLint was complaining here. If I am not mistaken then it is possible to drop the parentheses completely. Seems weird but python interprets this as a generator then.
```suggestion
                    if not all(isinstance(item, str) for item in generic_adder_cfg[field]):
```